### PR TITLE
fix: added Edit List button to all list pages

### LINKED ISSUE #2123: Bug: Add Edit List button to all views on List Feature
### Type of feature

🍕 Feature

### Current behavior

Right now, there's only an edit list button on the Overview tab.

list dashboard

### Suggested solution

- Add Edit List button to the Activity and Contributors tab as well. 

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by nickytonline**
## Description

This was initally omitted for the Product Hunt launch, but should have been added to all pages as part of #1977. Now all list pages have the Edit List button.

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents

Fixes #2123

## Mobile & Desktop Screenshots/Recordings

CleanShot 2023-11-13 at 11 45 26

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

File: pages/lists/[listId]/activity.tsx [hunk 1]
```diff
@@ -80,14 +83,14 @@ export const getServerSideProps = async (ctx: GetServerSidePropsContext) => {
         topContributor: mostActiveData?.data?.length ? mostActiveData.data[0] : null,
         projectData: projectData ?? [],
       },
+      isOwner: list && list.user_id === userId,
```
Comment (nickytonline) [inline pages/lists/[listId]/activity.tsx:86]: This is the same logic used on the Overview page.

Comment (nickytonline) [inline pages/lists/[listId]/activity.tsx:86]: This is the same logic used on the Overview page.

Comment (BekahHW) [reply pages/lists/[listId]/activity.tsx:86]: Thanks, @nickytonline !!!

**Review by BekahHW — APPROVED** [review]

**Review by OgDev-01 — APPROVED** [review]
Way to go!!! 🍕🍕🍕

**Pull request merged**